### ISSUE TITLE
Invalid arguments in the three.js loader when using compression

### ISSUE THREAD

**Issue by benbro** [KhronosGroup/glTF#223]
I'm converting the duck example with compression:

```
./collada2gltf -f duck.dae -c Open3DGC -m binary
```

FF gives me this error:

```
Error: invalid arguments
return new Uint16Array(resource, 0, ctx.indices.count);
glTFLoader.js (line 191)
```

**Comment by pjcozzi** [comment]
The compression extension is being worked in #398.  I suggest we close this issue and create a roadmap page (where ever @tparisi thinks) for the Three.js glTF loader.

**Comment by tparisi** [comment]
Yes, close.

I believe we agreed that it's better to continue the feature work in this repo and then submit the occasional PR to Three.js.

**Comment by pjcozzi** [comment]
OK, I'll close once we have the Three.js glTF loader posted.

**Comment by JoeKnow** [comment]
Has there been any progress on this front ?

**Comment by tparisi** [comment]
@JoeKnow please check out all the latest in master... the Three.js loader is fully upgraded. Is the issue still relevant?

**Comment by pjcozzi** [comment]
Closing given the new three.js glTF loader.

**Comment by proteamer** [comment]
FYI: new three.js glTF loader can't handle `o3dgc` compression unfortunately.

**Comment by pjcozzi** [comment]
That is OK for now since o3dgc is not complete and not fully implemented anywhere AFAIK: #398
